internal/usecase/daemon/attachment_effect_gate.go

```
@@ -194,53 +188,68 @@ func (t *attachmentEffectTicket) endTransportSend() {
 // beginCurrentAttachmentEffect waits out a transition that already froze ac, then
 // admits the capability derived from the session registry. Handshakes use it
 // after Welcome has completed so a replacement blocked behind that send can
 // publish before readiness or first paint.
-func (ac *attachedClient) beginCurrentAttachmentEffect(sess *session, tr ports.Transport) (attachmentConnectionToken, *attachmentEffectTicket, bool) {
+func (ac *attachedClient) beginCurrentAttachmentEffect(sess *session, tr ports.Transport) (attachmentCapability, *attachmentEffect, bool) {
 	//nolint:contextcheck // This compatibility wrapper deliberately delegates with a fresh context; cancellable callers use the context variant below.
 	return ac.beginCurrentAttachmentEffectContext(context.Background(), sess, tr)
 }
 
-func (ac *attachedClient) beginCurrentAttachmentEffectContext(ctx context.Context, sess *session, tr ports.Transport) (attachmentConnectionToken, *attachmentEffectTicket, bool) {
+func (ac *attachedClient) beginCurrentAttachmentEffectContext(ctx context.Context, sess *session, tr ports.Transport) (attachmentCapability, *attachmentEffect, bool) {
```

golangci-lint (2.12.2)

[error] Non-inherited new context, use function like `context.WithXXX` instead

(contextcheck)